components/nimbus/build.rs

```diff
@@ -2,7 +2,22 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+use std::process::Command;
+
 pub fn main() {
     #[cfg(feature = "uniffi-bindings")]
     uniffi_build::generate_scaffolding("./src/nimbus.udl").unwrap();
+
+    Command::new("glean_parser")
```

Comment:
Do you see a future where we'd just have glean_parser as a dev dependency and just call a generate function like we do for uniffi? I'm just thinking about keeping the tool up to date, people not having PATH set, etc etc.

(ah - I see you mention you are having problems doing that, but I'm happy to see that's the intent!)

Reply:
Yes, what I would really dream for is to update glean_parser to be written in Rust (or maybe partially replaced by UniFFI since they both accomplish the same thing, take a human-readable file and generate some code in various languages). That way it would be easy to include it as a dependency in the build and trigger it like `generate_scaffolding`.

Reply:
In the future this will probably get easier...   like once this gets more time and love: https://github.com/badboy/glean-parser